endpoints/openrtb2/video_auction.go

```diff
@@ -29,14 +30,14 @@ import (
 
 var defaultRequestTimeout int64 = 5000
 
-func NewVideoEndpoint(ex exchange.Exchange, validator openrtb_ext.BidderParamValidator, requestsById stored_requests.Fetcher, cfg *config.Configuration, met pbsmetrics.MetricsEngine, pbsAnalytics analytics.PBSAnalyticsModule, disabledBidders map[string]string, defReqJSON []byte, bidderMap map[string]openrtb_ext.BidderName, categories stored_requests.CategoryFetcher) (httprouter.Handle, error) {
+func NewVideoEndpoint(ex exchange.Exchange, validator openrtb_ext.BidderParamValidator, requestsById stored_requests.Fetcher, cfg *config.Configuration, met pbsmetrics.MetricsEngine, pbsAnalytics analytics.PBSAnalyticsModule, disabledBidders map[string]string, defReqJSON []byte, bidderMap map[string]openrtb_ext.BidderName, categories stored_requests.CategoryFetcher, videoFetcher stored_requests.Fetcher) (httprouter.Handle, error) {
```

Comment:
One minor suggestion: Can we maybe consider putting all the fetchers next to each other in the method parameter list for this and the other auction and amp endpoint methods? Not that it affects the execution of the code but I think it will really improve readability.

Probably the same also in the endpointDeps struct

Reply:
Makes sense